admin/models/school.js

```diff
@@ -5,9 +5,9 @@ const Schema = mongoose.Schema
 const randomGenerator = require('../lib/random-generator')
 
 const School = new Schema({
-  _id: { type: Number },
+  _id: {type: Number},
   leaCode: {
-    type: Number,
+    type: Schema.Types.Number,
```

Comment:
The approach I took was to diff the pupil and school models and copy the changes from pupil to admin, and then overwrite the pupil model with the more substantial admin model.

So these changes will have come from differences between the admin model and the pupil model.